iamctl/cmd/cli/importAll.go

```diff
@@ -41,6 +42,7 @@ var importAllCmd = &cobra.Command{
 		}
 		identityproviders.ImportAll(inputDirPath)
 		applications.ImportAll(inputDirPath)
+		claims.ImportAll(inputDirPath)
```

Comment:
We should recheck the order of importing these resources. IMO claims should be imported first, incase there are new claims configured in idps or apps.

Reply:
Re-ordered the importing statements.